<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>server</class>
 <widget class="QWidget" name="TwitchGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TwitchGUI</string>
  </property>


  <widget class="QLabel" name="background_image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../soy.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>

  <widget class="QLabel" name="user_list">
    <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
    </property>
    <property name="geometry">
       <rect>
          <x>0</x>
          <y>0</y>
          <width>1280</width>
          <height>720</height>
       </rect>
    </property>
    <property name="alignment">
      <set>Qt::AlignRight | Qt::AlignTop</set>
    </property>
  </widget>

  <widget class="QLabel" name="cmd_label">
    <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
    </property>
    <property name="geometry">
       <rect>
          <x>0</x>
          <y>0</y>
          <width>1280</width>
          <height>720</height>
       </rect>
    </property>
    <property name="alignment">
      <set>Qt::AlignCenter | Qt::AlignBottom</set>
    </property>

  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
